wiki/en/Running-a-Server.md

```diff
@@ -36,84 +37,101 @@ However, various problems can also arise when setting up Servers - especially wh
 
-Your Server will be listed in a Directory. By default, Jamulus has a list of built-in Directories that Clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. You can also have your Server listed on a Custom Directory, if that better meets your needs.
+To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. 
 
-<figure>
-	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Registered Server">
-<figcaption>How Registered Servers work</figcaption>
-</figure>
+If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 debian-based device), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines.
+
+1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) (or for Raspberry Pi etc. as above)
+1. Make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package for your architecture, for example: 
+
+	`sudo apt install ./{{ site.download_file_names.deb-headless }}`
+
+1. Enable the headless Server process: 
+
+	`sudo systemctl enable jamulus-headless`
+
+1. Add your desired [command line options](Running-a-Server#configuration) to the `ExecStart` line in the systemd service file (by default you will be running an Unregistered Server):
+
+	`sudo systemctl edit --full jamulus-headless`
 
-### 2. Unregistered
+1. Reload the systemd files and restart the headless Server:
+
+	`sudo systemctl daemon-reload && systemctl restart jamulus-headless` 
+
+_To upgrade your Server, just repeat the steps above._
+
+---
+
+## Server Types
+
+Servers can be run in one of three modes (either at home or on a 3rd party host), depending on your needs.
+
+### 1. Unregistered
 
 This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a Server.
 
+**For information about running an Unregistered Server [see this guide](Unregistered-Servers).**
+
 <figure>
 	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between Clients within a Jamulus Unregistered Server">
```

Comment:
Although only slightly related, I think we could/should put these "how things work" into the main repo

Reply:
Not sure I understand why (and if it means we have to maintain the same information in two places, that's not a good idea really)